src/main/java/com/synopsys/integration/alert/workflow/scheduled/PhoneHomeTask.java

```diff
@@ -182,26 +181,24 @@ private PhoneHomeService createPhoneHomeService(ExecutorService phoneHomeExecuto
         return channelMetadata.toArray(String[]::new);
     }
 
-    private Set<String> retrieveChannelMetadataForJobs(List<ConfigurationJobModel> jobs) {
+    private Set<String> retrieveChannelMetadataForJobs(List<DistributionJobModel> jobs) {
         Map<String, Integer> createdDistributions = new HashMap<>();
         String successKeyPart = "::Successes";
-        for (ConfigurationJobModel job : jobs) {
-            for (ConfigurationModel configuration : job.getCopyOfConfigurations()) {
-                String channelName = configuration.getField(ChannelDistributionUIConfig.KEY_CHANNEL_NAME).flatMap(ConfigurationFieldModel::getFieldValue).orElse("");
-                String providerName = configuration.getField(ChannelDistributionUIConfig.KEY_PROVIDER_NAME).flatMap(ConfigurationFieldModel::getFieldValue).orElse("");
-
-                if (StringUtils.isBlank(channelName) || StringUtils.isBlank(providerName)) {
-                    // We want to specifically get the channel configuration here and the only way to determine that is if it has these fields.
-                    continue;
-                }
+        for (DistributionJobModel job : jobs) {
+            String channelName = job.getChannelDescriptorName();
+            String providerName = new BlackDuckProviderKey().getUniversalKey();
+
+            if (StringUtils.isBlank(channelName)) {
```

Comment:
I was unsure if this is still necessary.

Reply:
The database does not allow `null` values for the channel descriptor name, so I'm inclined to think that this being blank would be a bug (if we are correctly validating the data that gets saved). So I think this can be removed.